core/src/services/lakefs/backend.rs

```diff
@@ -229,7 +230,7 @@ impl Access for LakefsBackend {
                 let decoded_response: LakefsStatus =
                     serde_json::from_reader(bs.reader()).map_err(new_json_deserialize_error)?;
 
-                meta.set_content_length(decoded_response.size_bytes);
+                meta.set_content_length(decoded_response.size_bytes.unwrap());
```

Comment:
If `size_bytes` can be optional, how about handling it properly instead of using unwrap?